### DISCUSSION TITLE
Automated language translation

### DISCUSSION THREAD

**Discussion by makeitr3al** [GitbookIO/community#249] [Feature Requests]
I'd love to have my content automatically translated by DeepL, Google or something else.

Now I'm trying to do that by using a GitHub repository which clones and translates original content after each change.

**Comment by addisonschultz** [comment]
Hey @makeitr3al, nice idea - have you gotten something to work with the GitHub repository workflow? There's been other discussions on similar topics as well - https://github.com/orgs/GitbookIO/discussions/225

**Reply by makeitr3al** [reply]
Thank you SO MUCH for your reply. I found the real game changer to realize my idea.

I used https://github.com/3ru/gpt-translate

**Reply by addisonschultz** [reply]
So happy  you got this up and running!
